041-VirtualWAN/Coach/01-any_to_any.md

```diff
@@ -0,0 +1,117 @@
+# Challenge 1: Any-To-Any connectivity - Coach's Guide
+
+**[Home](./README.md)** - [Next Challenge >](./02-vpn.md)
+
+## Notes and Guidance
+
+- You might want to pre-create VNets and VMs previous to the event to gain time
+
+## Solution Guide
+
+The following sections contain a possible solution based on Azure CLI
+
+### Creating RG, VWAN and hubs
+
+<details><summary>Code</summary>

```

Comment:
Really the only thing I can find in your hack that doesn't exactly sit well with me are the code snippet in the coach guide.

Besides the fact that html in markdown is a pet peeve of mine, I can't see any reason why we don't just remove the html and have the triple back tick code block always shown?

Alternate, we can break out this code into scripts and templates in Coach/Solutions folder... but this is really only useful if you want coaches to run the stuff directly and not give some explanation.

So I'll let you pick, my choice is the first one, just leave them as code blocks and remove the html so they are always visible.

Reply:
Yeah, I had that from the time when I had a single coach guide, to make it more readable. Now that it is split in challenges, fully agree with you. Let me remove that ugly HTML...

Reply:
Done!